sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptionsFactory.java

```diff
@@ -348,7 +348,7 @@ static boolean printHelpUsageAndExitIfNeeded(ListMultimap<String, String> option
             getRegisteredOptions(),
             new Predicate<Class<? extends PipelineOptions>>() {
               @Override
-              public boolean apply(Class<? extends PipelineOptions> input) {
+              public boolean apply(@Nonnull Class<? extends PipelineOptions> input) {
```

Comment:
[Guava Predicate](https://github.com/google/guava/blob/master/guava/src/com/google/common/base/Predicate.java#L80) defines `apply(@Nullable T input)`, so the `@Nullable` annotation is carried forward to override implementations. I've added `@Nonnull` to override the default interpretation.

This is an instance of FindBugs not playing well with 3rd-party libraries (Guava). An alternative would be to add a `@SuppressFBWarning` annotation, and explain why it's needed. I went with `@Nonnull` as it provides standard metadata that other tooling (IDEs) can take advantage of, and also because it is less distracting to the intent of the code than a full '@SuppressFBWarning` annotation.